tests/test_docs.py

```diff
@@ -73,14 +74,14 @@ def now(cls, *args, **kwargs):
 
 
 @pytest.mark.skipif(bool(skip_reason), reason=skip_reason or 'not skipping')
-@pytest.mark.parametrize('example', find_examples('docs', skip=sys.platform == 'win32'), ids=str)
+@pytest.mark.parametrize('example', find_examples(str(DOCS_ROOT), skip=sys.platform == 'win32'), ids=str)
```

Comment:
we should update `find_examples` to take a `Path`, it's probably just an annotation change required.

Reply:
Created PR at https://github.com/pydantic/pytest-examples/pull/6, but I'll merge this with the str cast for now